daemon/src/maker_cfd.rs

```diff
@@ -44,10 +44,6 @@ use xtras::SendAsyncSafe;
 
```

Comment:
re commit description: I'd suggest softer phrasing than "we definitely have a deadlock" when we are do not have 100% clarity due to incomplete logs